Riot/Modules/Home/AllChats/AllChatsLayoutSettingsManager.swift

```diff
@@ -96,4 +104,24 @@ final class AllChatsLayoutSettingsManager: NSObject {
             }
         }
     }
+    
+    // MARK: - Private
+    
+    private func track(activeFilters: AllChatsLayoutFilterType?) {
+        guard let activeFilters = activeFilters else {
+            Analytics.shared.updateUserProperties(allChatsActiveFilter: nil)
+            return
+        }
+
+        switch activeFilters {
+        case .unreads:
+            Analytics.shared.updateUserProperties(allChatsActiveFilter: .unreads)
+        case .favourites:
+            Analytics.shared.updateUserProperties(allChatsActiveFilter: .favourites)
+        case .people:
+            Analytics.shared.updateUserProperties(allChatsActiveFilter: .people)
+        default:
+            Analytics.shared.updateUserProperties(allChatsActiveFilter: .all)
```

Comment:
Same here, let's replace the default with .all. I like exhaustive switches 😄

Reply:
Actually it's not exhaustive. We have a `rooms` case that we don't use anymore for the moment.

Reply:
Oh okay, my bad 👍